File: src/main/java/com/durhack/sharpshot/util/ErrorBox.java
package com.durhack.sharpshot.util;

import javafx.scene.control.Alert;

public class ErrorBox {
    public static void alert(String title, String header, String content) {
        Alert alert = new Alert(Alert.AlertType.ERROR);
        alert.setTitle(title);
        alert.setHeaderText(header);
        alert.setContentText(content);
        alert.showAndWait();
    }
}

File: src/main/java/com/durhack/sharpshot/util/ErrorBox.kt
package com.durhack.sharpshot.util

import javafx.scene.control.Alert

object ErrorBox {
    fun alert(title: String, header: String, content: String) {
        val alert = Alert(Alert.AlertType.ERROR)
        alert.title = title
        alert.headerText = header
        alert.contentText = content
        alert.showAndWait()
    }
}
